docs/common/utilities.ts

```diff
@@ -43,7 +43,7 @@ export const replaceVersionInUrl = (url: string, replaceWith: string) => {
 };
 
 export const getVersionFromUrl = (url: string) => {
-  return url.split('/')[2];
+  return new URL(url, 'https://docs.expo.dev').pathname.split('/')[2];
```

Comment:
This didn't handle urls with query parameters, like `https://docs.expo.dev/versions/v44.0.0/sdk/notifications?redirected=`. 🤷